examples/tutorials/get-started/quickstart-azure.ipynb

```diff
@@ -426,7 +426,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.17"
+   "version": "3.8.13"
```

Comment:
@crazygao I remember you mentioned we need to keep this unchanged? but I forget the reason...

Reply:
No special reasons, we don't need to change this property.

Reply:
good catch, reverted